### PR TITLE
[Merged by Bors] - Update to 3.0.21-alpha

### PR DESCRIPTION
**Pull request by eyalb181**

### CODE REVIEW

**Review by aviramha — APPROVED** [review]

**Comment by aviramha** [comment]
bors cancel

**Review by infiniteregrets — COMMENTED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+## 3.0.21-alpha
```
[the remark this answers is not in the document]
Comment (eyalb181) [reply CHANGELOG.md:10]: Fixed, thanks!

**Pull request closed without merging**